Cargo.toml

```diff
@@ -13,7 +13,9 @@ rust-version  = "1.65"
 [workspace.dependencies]
 cosmwasm-schema = "1.2.1"
 cosmwasm-std    = "1.2.1"
-cw2             = "1.0.1"
+# FIXME: change back to the "official" crates.io release after this PR is merged:
+# https://github.com/CosmWasm/cw-plus/pull/858
+cw2             = { git = "https://github.com/mars-protocol/cw-plus", rev = "1a3a944" }
```

Comment:
Seems to be an overkill just to get the `assert_contract_version` helper: https://github.com/CosmWasm/cw-plus/pull/858/files#diff-a35715fa1fde56c42db0d8f2845ebc6a571a801699c0705f0080237820d96ee6R59-R83

Would suggest just to just put it in this contract until we have an "official" version of it.

Reply:
@Art3miX My PR to cw2 referenced in the comment has been merged. Now we can use the main branch of cw2 instead of mars-protocol fork. Imo this is more elegant than including extra code in this repo which we need to remove later.

Reply:
Not sure what's going on here but when I go to the main branch of cw2 I am not seeing the assert_contract_version that is referenced here.